js/components/EditProfileForm/EditProfileForm.js

```diff
@@ -98,19 +142,16 @@ export default class EditProfileForm extends Component {
               />
             </View>
             <TouchableOpacity
-              activeOpacity={0.6}
               style={styles.confirmButton}
-              onPress={() => {
-                navigation.navigate("Profile");
-              }}
+              onPress={() => handleSubmit()}
             >
               <LinearGradient
-                 start={{ x: 0, y: 0 }}
-                 end={{ x: 0, y: 1 }}
-                 colors={[
-                   globalStyles.blueGradientColor.start.color,
-                   globalStyles.blueGradientColor.end.color
-                 ]}
+                start={{ x: 0, y: 0 }}
+                end={{ x: 0, y: 1 }}
+                colors={[
+                  globalStyles.blueGradientColor.start.color,
+                  globalStyles.blueGradientColor.end.color
+                ]}
                 style={[
                   StyleSheet.absoluteFill,
                   {
```

Comment:
use the styles.js file to do your styles and use Dimensions.

Reply:
I was talking about height, width and other styles. colors is a prop so you can not put it in the styles.js file.